tests/coverage/coverage.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package coverage
+
+import (
+	"github.com/skydive-project/skydive/cmd"
+)
+
+func skdyiveCoverage() {
+	cmd.RootCmd.Execute()
```

Comment:
this file seems to be useless ? This line could be used in the Test file

Reply:
it's a trick, because you cannot have a package without a source file and only a test file